src/clj/rems/api/services/resource.clj

```diff
@@ -59,5 +59,5 @@
                                     :archived archived})
         {:success true}))))
 
-(defn get-resource [id] (resource/get-resource id))
-(defn get-resources [filters] (resource/get-resources filters))
+(defn get-resource [id user-id] (resource/get-resource id user-id))
+(defn get-resources [filters user-id] (resource/get-resources filters user-id))
```

Comment:
could user be in filters?

Reply:
Not sure, I looked at db/apply-filters and it seems to work by checking that certain key(s) exist in the given map, and filter out those elements for which they don't. I couldn't quite see how to implement the filtering by certain organization in that framework.

Reply:
you're probably right, `filters` has a specific meaning in our db code

Reply:
Key value filter would make sense as type though

Reply:
I think it matters less now that user-id will not be passed separately as a parameter.